plugins/webhooks/index.js

```diff
@@ -413,67 +447,58 @@ function pullRequestEvent(pluginOptions, request, reply, parsed) {
 
     request.log(['webhook', hookId], `PR #${prNum} ${action} for ${fullCheckoutUrl}`);
 
-    // Fetch the pipeline associated with this hook
-    return obtainScmToken(pluginOptions, userFactory, username, scmContext)
-        .then((token) => {
-            scmConfig.token = token;
-
-            return pipelineFactory.scm.parseUrl({
-                checkoutUrl: fullCheckoutUrl,
-                token,
-                scmContext
-            });
-        })
-        .then((scmUri) => {
-            scmConfig.scmUri = scmUri;
+    return pipelineFactory.scm.parseUrl({
+        checkoutUrl: fullCheckoutUrl,
+        token,
+        scmContext
+    }).then((scmUri) => {
+        scmConfig.scmUri = scmUri;
```

Comment:
do you need to pass `token` into `scmConfig` as well?
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/webhooks/index.js#L409

Reply:
It's needed... Passed.